scripts/CMakeLists.txt

```diff
@@ -29,18 +29,18 @@ ov_cpack_add_component(${OV_CPACK_COMP_SETUPVARS} HIDDEN)
 if(UNIX)
     set(_setupvars_file_full_path "${CMAKE_CURRENT_SOURCE_DIR}/setupvars/setupvars.sh")
 elseif(WIN32)
+    # setupvars.bat
     set(_setupvars_file "setupvars/setupvars.bat")
     set(_setupvars_file_full_path "${CMAKE_CURRENT_SOURCE_DIR}/${_setupvars_file}")
-    if(USE_BUILD_TYPE_SUBFOLDER AND CMAKE_BUILD_TYPE AND NOT CMAKE_BUILD_TYPE MATCHES "^(Debug|Release)$")
-        # Patch primary configuration in setupvars.bat which is "Release" by default.
-        # Note setupvars secondary configuration is always "Debug".
-        message(STATUS "Patching content of ${_setupvars_file_full_path} for CMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}")
-        file(READ "${_setupvars_file_full_path}" _setupvars_content)
-        string(REPLACE "Release" ${CMAKE_BUILD_TYPE} _setupvars_content "${_setupvars_content}")
-        set(_setupvars_file_full_path "${OpenVINO_BINARY_DIR}/${_setupvars_file}")
-        message(STATUS "Writing patched content to ${_setupvars_file_full_path}")
-        file(WRITE "${_setupvars_file_full_path}" "${_setupvars_content}")
-    endif()
+
+    # setupvars.ps1
+    set(_setupvars_file_pwsh "setupvars/setupvars.ps1")
+    set(_setupvars_file_pwsh_full_path "${CMAKE_CURRENT_SOURCE_DIR}/${_setupvars_file_pwsh}")
+
+    install(PROGRAMS "${_setupvars_file_pwsh_full_path}"
+            DESTINATION .
+            COMPONENT ${OV_CPACK_COMP_SETUPVARS}
+            ${OV_CPACK_COMP_SETUPVARS_EXCLUDE_ALL})
```

Comment:
we can combine installation of several files into a single command and pass a list of files as input